### ISSUE TITLE
compact usage examples component

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#350]
we need a compact version of the usage examples component for small devices. 

I don't know what the best design is here, but the most important data to have displayed is the usage example text itself. Maybe we could have the citation information show in a mouseover popup?  And the target word be the active link to the source text?

Feel free to experiment a bit with this.

**Comment by irina060981** [comment]
I have created layout for compact and desktop version.
Here it is screenshots:
**desktop** 
(it is responsive by the width of the panel)
![image](https://user-images.githubusercontent.com/12377640/54197542-f0e6e300-450f-11e9-92d1-372559ced4ce.png)

**mobile** (browser emulator)
![image](https://user-images.githubusercontent.com/12377640/54197970-0ad4f580-4511-11e9-813b-ea381d78140e.png)

**mobile** (phone)
![image](https://user-images.githubusercontent.com/12377640/54198460-44f2c700-4512-11e9-8e53-db6a00430d1e.png)

----------------------------------
But there are some strange behaviour in compact panel - it scale font inside panel compact according to screen width. I didn't find how it works. And for now - I have increased font inside my tab.
Other tabs look with very small font.
I have told also @kirlat .
I didn't find out myself - what is going on with font size inside panel.
